apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -202,6 +222,12 @@ export default {
   async mounted() {
     this.loaded = false;
     this.client = Client.newClient(this.graphqlURL);
+    const settings = await this.client.fetchSettings(this.graphqlURL);
+
+    this.useChapters =
+      settings.find((item) => item.key === IS_CHAPTERS_ENABLED)?.value !=
```

Comment:
Might be nice to have a getSetting(settings, settingName) util function since we do that in several places.

Reply:
Will not be done in this pr